forest/src/cli/mod.rs

```diff
@@ -224,6 +217,31 @@ impl CliOpts {
     }
 }
 
+pub fn find_default_config() -> Option<Config> {
+    // check forest config path first, then look at other default locations
+    let path = if let Ok(config_file) = std::env::var("FOREST_CONFIG_PATH") {
+        PathBuf::from(config_file)
+    } else if let Ok(root_config_path) = std::env::var("XDG_CONFIG_HOME") {
+        // determine if XDG_CONFIG_HOME is set for default locations, otherwise use home_dir
+        PathBuf::from(root_config_path + "/forest/config.toml")
+    } else {
+        // $XDG_CONFIG_HOME does not exist, check ~/.forest.toml
+        PathBuf::from(get_home_dir() + "/.forest/config.toml")
```

Comment:
Would be better not to use strings to craft paths (see `Path` and `PathBuff` types).

Reply:
:+1: